### ISSUE TITLE
Create brew recipe for upstream version; with access to registration-free bundle (download)

### ISSUE THREAD

**Issue by gbraad** [code-ready/crc#1234]
This depends on #977 

When a registration-free bundle can be download, we can implement a way to download imagebundles.

**Comment by praveenkumar** [comment]
@gbraad do we still want the brew recipe, I think as soon as we get some location from upstream side, we should just host it for public consumption.

**Comment by gbraad** [comment]
right, but this is when an OKD release or downloadable OCP images are available separate from the binary. This is currently not the case
